_includes/wings.html

```diff
@@ -0,0 +1,21 @@
+<div class="container">
+  <div class="row">
+    {% for wing in site.data.wings %}
+      <div class="square" id="sq{{ wing.name }}">
+        <h3>{{ wing.name }}</h3>
+        <p>
+          {% if wing.website %}
+            <a href="{{ wing.website }}">&#128279;</a>
```

Comment:
yeah RIP... I didn't spend too much time playing around with them bc it was the last thing I did